stable_baselines/her/her.py

```diff
@@ -39,11 +74,12 @@ def learn(self, total_timesteps, callback=None, seed=None, log_interval=100, tb_
         return self.model.learn(total_timesteps, callback=None, seed=None, log_interval=100, tb_log_name="HER",
```

Comment:
to avoid future headaches, did you mean
 ```python
return self.model.learn(total_timesteps, callback=callback, seed=seed, log_interval=log_interval, tb_log_name=tb_log_name, reset_num_timesteps=reset_num_timesteps, replay_wrapper=self.replay_wrapper)
```
?

Reply:
oui ^^#

thanks

Reply:
done